stubs/app/server/lib/globalConfig.ts

```diff
@@ -0,0 +1,19 @@
+import path from "path";
```

Comment:
So this code is in `stubs`. Do we need to be ready to add something in all other flavors of Grist? -ee, -saas, -desktop, -saas?

One way to add a method without breaking all other builds is to add it to the ICreate interface as an optional method, like checkBackend?() or configure?(). Then any Grist versions we haven't gotten to would still work, either with no change or inherited behavior from core, depending on their implementation of create.

Reply:
Update: after a conversation, seems ok to go ahead as is. May or may not be a fix-up needed in grist-saas, which we can do.